api/src/http.rs

```diff
@@ -106,7 +106,7 @@ pub struct BlobCacheList {
 
 #[derive(Clone, Deserialize, Debug)]
 pub struct BlobObjectParam {
-    pub blob_id: String,
+    pub domain_id: String,
```

Comment:
It would be better to match blobs by both domain_id and blob_id. 
It means matching all blobs with a common domain_id prefix when the blob_id is empty.